File: pset6/mario.py
import cs50

def PrintRow(height):
    inc = height
    dec = height

    # setup number of spaces and 'stairs'
    for i in range(0, height):
        inc += 1
        dec -= 1

        for z in range(1, (dec + 1)):
            print(" ", end= "")

        for y in range(height, (inc + 1)):
            print("#", end= "")

        print("")

def main():

    # establish do-while
    while True:
        print("Height: ", end= "")
        h = cs50.get_int();
        if(h < 24 or h > -1):
            break;

    PrintRow(h)

if __name__ == "__main__":
    main()

File: pset1/mario.c
#include <stdio.h>
#include <cs50.h>

void PrintRow(int height);

int main(void) {
    
    int x;
    
    do {
        printf("Height: ");
        x = get_int();
    } 
    while (x < 0 || x >23);
    
    PrintRow(x);
    
}

void PrintRow(int height) {
    
        int increasing = height;
        int decreasing = height;
        
    
        for( int i = 0; i < height; i++) {
        
        increasing += 1;
        decreasing -= 1;
        
        //print the spaces (decreasing iteration)
       for ( int z = 1; z <= decreasing; z++) {
           
           printf(" ");
    
        }
        
         //print the hashes (increasing iteration)
        for ( int y = increasing ; y >= height; y--) {
       
         printf("#");
        
       }
      
      //print a new line
        printf("\n");
        
        
    }
   
    
}

